src/test/test-boot-timestamps.c

```diff
@@ -91,5 +91,9 @@ int main(int argc, char* argv[]) {
         r = test_boot_timestamps();
         assert(r >= 0);
 
-        return (p > 0 || q > 0 || r >> 0) ? EXIT_SUCCESS : EXIT_TEST_SKIP;
+        bool any = p > 0 || q > 0 || r > 0;
```

Comment:
Note this seems to be fixing a typo here, `|| r >> 0)` in the preceding, while this one has `|| r > 0)`. This looks intentional to me. Probably good to add it to the commit description, just to make sure.

Also, I think code would be slightly better as:

```c
if (!(p > 0 || q > 0 || r > 0)) {   // or if (!any) also fine...
    log_notice("%s: no tests are possible.", program_invocation_short_name);
    return EXIT_TEST_SKIP;
}

return EXIT_SUCCESS;
```

Perhaps adding "skipping" to the log_notice message would also be good.

Reply:
> Perhaps adding "skipping" to the log_notice message would also be good.

:+1: